src/main/java/com/redhat/devtools/lsp4ij/client/features/EditorBehaviorFeature.java

```diff
@@ -99,6 +99,17 @@ public boolean isEnableTextMateNestedBracesImprovements(@NotNull PsiFile file) {
         return false;
     }
 
+    /**
+     * Whether or not the semantic tokens-based file view provider is enabled.
+     *
+     * @param file the file
+     * @return true if the semantic tokens-based file view provider is enabled; otherwise false
+     */
+    public boolean isEnableSemanticTokensFileViewProvider(@NotNull PsiFile file) {
```

Comment:
Feature flag disabled by default for non-user-defined language server definitions.